.github/workflows/integration.yml

```diff
@@ -16,6 +16,8 @@ env:
   CARGO_TERM_COLOR: always
   CARGO_INCREMENTAL: 0
   S3_BUCKET_NAME: ${{ vars.S3_BUCKET_NAME }}
+  S3_EXPRESS_ONE_ZONE_BUCKET_NAME: ${{ vars.S3_EXPRESS_ONE_ZONE_BUCKET_NAME }}
+  S3_EXPRESS_ONE_ZONE_ENDPOINT: ${{ vars.S3_EXPRESS_ONE_ZONE_ENDPOINT }}
```

Comment:
Can we ditch this now?

New Rust SDK has been released with S3 Express support! https://github.com/awslabs/aws-sdk-rust/releases/tag/release-2023-11-28.2

Reply:
ah, in that case yes. I will open another PR to update the SDK and our tests.